apps/src/lib/ui/SettingsCog.jsx

```diff
@@ -31,10 +33,9 @@ export default class SettingsCog extends Component {
     managingModels: false
   };
 
-  targetPoint = {top: 0, left: 0};
-
   open = () => this.setState({open: true});
   close = () => this.setState({open: false});
+  toggleOpen = () => this.setState({open: !this.state.open});
```

Comment:
This toggle feature may be nice to add to the Java Lab tab menu.

Reply:
Do you mean this one? Seems like a bug that it doesn't close when I click the "up" arrow:

<img width="222" alt="image" src="https://user-images.githubusercontent.com/25372625/205187451-b6695819-94f5-4ae5-9bfe-a94b2c719c44.png">

Reply:
Yes - agree it should close when 'up' is closed.